elephas/spark_model.py

```diff
@@ -22,7 +22,7 @@
 from .optimizers import SGD as default_optimizer
 
 from keras.models import model_from_yaml
-from keras import constraints
+from keras.constraints import Constraint
```

Comment:
you never seem to use `Constraint`, am I missing something?

Reply:
This is just in case someone would like to use real constraints in the future (the import refers to the right place)

Reply:
I see what happened, I'll remove it

Reply:
ok, just let me know. I'll merge after